### PR TITLE
Migeng 71

### PR DESCRIPTION
**Pull request by m-brophy**
https://jira.coreos.com/browse/MIGENG-71

### CODE REVIEW

File: src/test/java/org/jboss/xavier/analytics/test/RHVAdditionalContainerCapacityTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        commands.add(CommandFactory.newFireAllRules(NUMBER_OF_FIRED_RULE_KEY));
+        // last create the command to retrieve the objects available in
+        // the working memory at the end of the rules' execution
+        commands.add(CommandFactory.newGetObjects(GET_OBJECTS_KEY));
+
+        // execute the commands in the KIE session and get the results
+        Map<String, Object> results = Utils.executeCommandsAndGetResults(kieSession, commands);
+
+        // check that the number of rules fired is what you expect
+        Assert.assertEquals(2, results.get(NUMBER_OF_FIRED_RULE_KEY));
```
Comment (mrizzi) [inline src/test/java/org/jboss/xavier/analytics/test/RHVAdditionalContainerCapacityTest.java:72]:
```suggestion
        Assert.assertEquals(2, results.get(NUMBER_OF_FIRED_RULE_KEY));
        Utils.verifyRulesFiredNames(this.agendaEventListener, "AgendaFocusForTest", "RHVAdditionalContainerCapacity");
```

**Review by mrizzi — COMMENTED** [review]
The rule looks good.
I would just add the check for the names of the fired rules in the test and then it will be perfect :wink:

**Comment by m-brophy** [comment]
@mrizzi your alteration has been committed to the PR now

**Review by mrizzi — APPROVED** [review]
Thanks @m-brophy, it works great :+1:

**Pull request merged**